packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -0,0 +1,542 @@
+                        columnSize = '22%'
+                        break
+                      case 'type':
+                        columnSize = '22%'
+                        break
+                      default:
+                        break
+                    }
+                    return (
+                      <th
+                        key={header.id}
+                        colSpan={header.colSpan}
+                        style={{
+                          width: columnSize,
+                          position: 'sticky',
+                          top: '0px',
+                          background: '#fff',
+                        }}
+                      >
+                        {header.isPlaceholder ? null : (
+                          <>
+                            <div
+                              {...{
+                                className: header.column.getCanSort()
+                                  ? 'SRC-hand-cursor'
+                                  : '',
+                                onClick:
+                                  header.column.getToggleSortingHandler(),
+                              }}
+                            >
+                              {flexRender(
+                                header.column.columnDef.header,
+                                header.getContext(),
+                              )}
+                              {{
+                                asc: (
+                                  <IconSvg
+                                    icon={'sortUp'}
+                                    wrap={false}
+                                    sx={{
+                                      color: 'primary.main',
+                                      backgroundColor: 'none',
+                                      float: 'right',
+                                      marginRight: '5px',
+                                    }}
+                                  />
+                                ),
+                                desc: (
+                                  <IconSvg
+                                    icon={'sortDown'}
+                                    wrap={false}
+                                    sx={{
+                                      color: 'primary.main',
+                                      backgroundColor: 'none',
+                                      float: 'right',
+                                      marginRight: '5px',
+                                    }}
+                                  />
+                                ),
+                              }[header.column.getIsSorted() as string] ?? null}
+                            </div>
+                            {header.column.getCanFilter() &&
+                            totalRowCount > UNMANAGEABLE_SUBJECT_COUNT ? (
+                              <div>
+                                <Filter column={header.column} table={table} />
```

Comment:
Love how simple it is to render this control! I think if we decide we want to align more closely with the design, it seems like this would be really easy to pull out of the table, as long as we extract the right column object!